argusclient/client.py

```diff
@@ -139,19 +142,27 @@ def __init__(self, argus):
 
 
 class BaseModelServiceClient(object):
-    def __init__(self, argus, get_all_path=None, get_all_params=None):
+    def __init__(self, argus, get_all_path=None,
+                                get_all_params=None,
+                                get_all_request_type=None,
+                                get_all_body=None):
```

Comment:
I have a concern on the growth of params here. How about encapsulating all of them into a `dict`:
```suggestion
    def __init__(self, argus, get_all_req_opts=None):
```

See the below change on how it gets interpreted.

Instead of a `dict`,  you may define a new concrete type, something like `ArgusRequest`, or at least create constants for the `dict` keys.